torch/_functorch/aot_autograd.py

```diff
@@ -1148,8 +1156,18 @@ def joint_helper(primals, tangents):
     def fwd_helper(*args):
         return functionalized_f_helper(*args)
 
+    helper = joint_helper if trace_joint else fwd_helper
+    if config.functionalize_rng_ops:
+        # Setup the wrapper for functionalization of rng ops
+        helper, args = create_functionalized_rng_ops_wrapper(helper, args, trace_joint)
+
     with enable_python_dispatcher():
-        return make_fx(joint_helper if trace_joint else fwd_helper, decomposition_table=aot_config.decompositions)(*args)
+        fx_g = make_fx(helper, decomposition_table=aot_config.decompositions)(*args)
+
+    if config.functionalize_rng_ops:
+        aot_config.philox_total_offsets = PhiloxStateTracker.get_accumulated_offsets()
```

Comment:
How is this a config?

Reply:
Adding comment

        # Total offsets differ for each AOT traced graph, so they can't be saved
        # in the PhiloxStateTracker singleton object. Therefore, we save them
        # into aot_config object as aot_config is alive in the runtime wrapper
        # returned by AOTAutograd (both autograd.Function for training and
        # simple wrapper for inference

If this seems very hacky, we can think of adding some new field in AOT Autograd that is available at both compile time and runtime.

Reply:
I guess I'm not familiar with aot_config :)
But the config name suggested to be that this was a shared object across invocations.

Reply:
I agree with Brian's suggestion; it should go in the Meta.

Reply:
Thanks @bdhirsh. This was very helpful!